File: onebot-sdk/src/main/java/cn/evole/onebot/sdk/response/guild/GuildServiceProfileResp.java
package cn.evole.onebot.sdk.response.guild;

import com.google.gson.annotations.SerializedName;
import lombok.Data;

/**
 * @author cnlimiter
 */
@Data
public class GuildServiceProfileResp {

    @SerializedName("nickname")
    private String nickname;

    @SerializedName("tiny_id")
    private String tinyId;

    @SerializedName("avatar_url")
    private String avatarUrl;

}

File: onebot/src/main/kotlin/sdk/response/guild/GuildServiceProfileResp.kt
package cn.evolvefield.onebot.sdk.response.guild

import com.google.gson.annotations.SerializedName

class GuildServiceProfileResp {
    @SerializedName("nickname")
    var nickname: String = ""

    @SerializedName("tiny_id")
    var tinyId: String = ""

    @SerializedName("avatar_url")
    var avatarUrl: String = ""
}
